trame_server/protocol.py

```diff
@@ -39,8 +39,17 @@ def configure(args):
             CoreServer.authentication_token = args.authKey
 
     @staticmethod
-    def server_start(options, **kwargs):
-        return server.start_webserver(options=options, protocol=CoreServer, **kwargs)
+    def server_start(
+        options, disable_logging=False, backend="aiohttp", exec_mode="main"
+    ):
+        # NOTE: **kwargs to wslink's start_webserver are unused
+        return server.start_webserver(
```

Comment:
ref https://github.com/Kitware/wslink/blob/052f86343ef0ac2828af7997c8ddfc4fcb9dad4f/python/src/wslink/server.py#L186

addition `**kwargs` are all ignored/unused in `wslink`

Reply:
The pb with that is if wslink add more args, we will need to update the trame-server as well. 
Better to get the kwargs freedom.

Reply:
Your call... I've fallen into `kwargs` traps like this too many times that I now avoid `kwargs` as much as possible since they bring up issues like this.

Flexibility at the cost of hard to track down issues